docs/content/configuration.md

```diff
@@ -864,12 +864,13 @@ It also represents the configuration of the inter-query _value_ cache that built
-| `caching.inter_query_builtin_value_cache.max_num_entries`                | `int` | No | Maximum number of entries in the Inter-query value cache. OPA will drop random items from the cache if this limit is exceeded. By default, set to `0` indicating unlimited size.                    |
+| Field                                                                    | Type    | Required | Description                                                                                                                                                                                                                                          |
+|--------------------------------------------------------------------------|---------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `caching.inter_query_builtin_cache.max_size_bytes`                       | `int64` | No       | Inter-query cache size limit in bytes. OPA will drop old items from the cache if this limit is exceeded. By default, no limit is set.                                                                                                                |
+| `caching.inter_query_builtin_cache.forced_eviction_threshold_percentage` | `int64` | No       | Threshold limit configured as percentage of `caching.inter_query_builtin_cache.max_size_bytes`, when exceeded OPA will start dropping old items permaturely. By default, set to `100`.                                                               |
+| `caching.inter_query_builtin_cache.stale_entry_eviction_period_seconds`  | `int64` | No       | Stale entry eviction period in seconds. OPA will drop expired items from the cache every `stale_entry_eviction_period_seconds`. By default, set to `0` indicating stale entry eviction is disabled.                                                  |
+| `caching.inter_query_builtin_value_cache.max_num_entries`                | `int`   | No       | Maximum number of entries in the Inter-query value cache. OPA will drop random items from the cache if this limit is exceeded. By default, set to `0` indicating unlimited size.                                                                     |
+| `caching.inter_query_builtin_value_cache.named.io_jwt.max_num_entries`   | `int`   | No       | Maximum number of entries in the `io_jwt` cache, used by the [`io.jwt` token verification](../policy-reference/#tokens) built-in functions. OPA will drop random items from the cache if this limit is exceeded. By default, this cache is disabled. |
```

Comment:
Should we add a note about the cache key, any other relevant details?

Reply:
This section explains that there is a separate cache used by the `io.jwt` built-ins that can be individually configured. Is there some other piece of information you'd like to see here?

About the cache key, are you referring to it's relative size compared to the cached value? I don't know if this implementation detail is actually relevant. If the size relationship was reversed, the same amount of data is still stored.

As a side note: the size of the cache entries could be improved by allowing users to add keys to the configuration, and pass their configured name instead of the complete key definition in their policies. This has other, key-management related, benefits, and might be something to consider for a future release.